### PR TITLE
[Feat] Logger using log4js lib in express.

### LINKED ISSUE #32: [Feat] Implement Logging system in server
We need to implement logger which supports debug, prod environment and should provide proper logging information for each actions.

### PR DESCRIPTION
**Pull request by vineetvk01**
Closes #32 

Implementing logger library in the server side for logging.
How to use :

`import Logger from './logger';`
`const logger = Logger('file_name');`
`logger.info('Provide` your message');`

### CODE REVIEW

**Review by nitishmittal1990 — APPROVED** [review]
LGTM

File: server/src/logger/logger.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import log4js from 'log4js';
+
+const logger = (identifier) => {
+    const logger = log4js.getLogger(identifier);
+    logger.level = 'info';
```
Comment (rishabh1403) [inline server/src/logger/logger.js]: expose at `error` type too. Don't hardcode everything as info :)

**Review by rishabh1403 — DISMISSED** [review]

File: server/src/logger/logger.js (hunk 1 above)
Comment (vineetvk01) [reply server/src/logger/logger.js]: Thats logger level. Everything above INFO Level will be printed as logs.
https://www.npmjs.com/package/log4js#usage

Comment (vineetvk01) [reply server/src/logger/logger.js]: TRACE < DEBUG < **INFO < WARN < ERROR < FATAL**

Comment (nimish-gupta) [reply server/src/logger/logger.js]: Store the 'info' variable in the env, and check whether its a dev or production or staging environment and show logs according to that.
Like in dev mode you might need the debug logs and traces.

**Review by nimish-gupta — APPROVED** [review]
LGTM, just resolve the following comment.

File: server/src/logger/logger.js (hunk 1 above)
Comment (vineetvk01) [reply server/src/logger/logger.js]: Okay!

**Pull request merged**